qdrant-landing/content/blog/qdrant-1.8.x.md

```diff
@@ -0,0 +1,130 @@
+---
+title: "Welcome to Qdrant 1.8.0!"
+draft: false
+slug: qdrant-1.8.x 
+short_description: "Look at what's new in Qdrant 1.8.0!"
+description: "Sparse vector performance, Text index loading optimization, Text immutability, Dynamic CPU saturation" 
+preview_image: /blog/qdrant-1.8.x/qdrant-1.8.0.png
+title_preview_image: /blog/qdrant-1.8.x/qdrant-1.8.0.png
+small_preview_image: /blog/qdrant-1.8.x/qdrant-1.8.0.png
+date: 2024-02-26T11:12:00-08:00
+author: Mike Jang
+featured: false 
+tags:
+  - vector search
+  - new features
+  - sparse vector performance
+  - dymamic CPU saturation
+  - text immutability
+  - text loading optimization
+weight: 0 # Change this weight to change order of posts
+---
+
+Today, we are pleased to announce the release of [Qdrant 1.8.0](https://github.com/qdrant/qdrant/releases/tag/v1.8.0).
+We've optimized performance in a number of ways. We've addressed issues with:
+
+<!-- Shard deltas (deferred to v1.9) -->
+- Sparse vectors
+- Dynamic CPU saturation
+- Text immutability
+- Text loading optimization
+
+We've also added <!-- TBD -->.
+
+Did we miss something? We rely on your feedback to drive our development. We
+welcome your contributions, especially in our [Discord community](https://qdrant.to/discord). Join us, introduce yourself, and help us build the best vector search engine!
+
+## Improved sparse vector performance
+
+The primary focus of Qdrant is performance. It's why we build our work in Rust.
+We also want to optimze performance in "real-world" situations.
+
+We used the [NeurIPS 2023 datasets](https://big-ann-benchmarks.com/neurips23.html). We've optimized our search to improve throughput by a factor of 16.
+
+We set up test benchmarks in our [Sparse vectors benchmark](https://github.com/qdrant/sparse-vectors-benchmark) repository. For our benchmarks, we use
+moderately-sized Azure instances with separate clients and servers:
+<!-- Arnaud isn't sure we should include these details. I think they're useful for our readers. (I've removed the Azure configuration bits described elsewhere.)  -->
```

Comment:
I slept on this and here is what I prefer.

We want the users to know that the sparse vector search got faster and that we got there by using a standardized dataset (using a link to our benchmark code).

In the spirit of conciseness, we could share the current graph by saying that it is an example of running the full query dataset against a 8 CPU machine.

Reply:
Thanks @agourlay . You've simplified things. I hope [these changes](https://github.com/qdrant/landing_page/pull/648/files#diff-c37269758c74910bcc8e4dae1c4ccc2fb994517b6ef01c4a5e2aa55109daaf27R44-R55) lays out what you're looking for.